Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Rebuild the source code only when needed
+FROM registry.access.redhat.com/ubi8/nodejs-14-minimal AS builder
+USER root
+WORKDIR /app
+RUN npm install -g yarn
+COPY . .
+RUN $(npm get prefix)/bin/yarn install --frozen-lockfile --ignore-optional
+RUN $(npm get prefix)/bin/yarn format
+RUN $(npm get prefix)/bin/yarn build
+RUN $(npm get prefix)/bin/yarn install --production --ignore-scripts --prefer-offline --ignore-optional
+
+# Production image, copy all the files and run next
+FROM registry.access.redhat.com/ubi8/nodejs-14-minimal AS runner
+USER root
+WORKDIR /app
+RUN microdnf install shadow-utils
+
+ENV NODE_ENV production
+
+RUN groupadd -g 1001 nodejs
+RUN useradd nextjs -u 1001
+
+COPY --from=builder /app/out ./out
```

Comment:
Could we simplify these copy statements by doing something like this?
```
COPY --from=builder /app ./
RUN chown nextjs:nodejs ./.next
```

Reply:
There isnt an easier way to copy the information because most of the data can be deleted